packages/client/src/navigation/index.ts

```diff
@@ -246,6 +248,10 @@ export function goToTeamUserList(id: string) {
   })
 }
 
+export function goToOrganizationList(index?: string) {
```

Comment:
Let's rename the `index` variable to something more descriptive like `locationId`, so it's easier for people reading this code to understand what the value should be